textile/features.textile

```diff
@@ -254,6 +254,13 @@ h3(#rest-auth). Auth
 ** @(RSA16b)@ If the library is provided with a token without the corresponding @TokenDetails@, then this holds a @TokenDetails@ instance in which only the @token@ attribute is populated with that token string
 ** @(RSA16c)@ Is set with the current token (if applicable) on instantiation and each time it is replaced, whether the result of an explicit @Auth#authorize@ operation, or a library-initiated renewal resulting from expiry or a token error response
 ** @(RSA16d)@ Is @null@ if there is no current token, including after a previous token has been determined to be invalid or expired, or if the library is using basic auth
+* @(RSA17)@ @Auth#revokeTokens@ function:
+** @(RSA17a)@ Takes a @TokenRevocationTargetSpecifier@ or an array of @TokenRevocationTargetSpecifier@s and sends them in a POST request to /keys/{API_KEY_NAME}/revokeTokens, with the newBatchResponse query param set to "true", where @API_KEY_NAME@ is the API key obtained from @AuthOptions#key@
```

Comment:
> `@API_KEY_NAME@ is the API key obtained from @AuthOptions#key@`

it's the key name only, not the full api key. (sdks already know what a key name is because the requestToken endpoint also uses the key name, and they need it to create a signed token request too)

Reply:
Updated. I've still kept some instructions on how to obtain the key name for clarity, since this doesn't exist elsewhere in the spec